plugins/org.eclipse.sirius.debug/src/org/eclipse/sirius/api/debug/AbstractDebugView.java

```diff
@@ -145,12 +135,12 @@ public void selectionChanged(IWorkbenchPart part, ISelection selection) {
     /**
      * Get the main object selected from the Eclipse ISelection.
      */
-    private Option<Object> getSelectedElement(ISelection selection) {
+    private Optional<Object> getSelectedElement(ISelection selection) {
         if (selection instanceof IStructuredSelection) {
```

Comment:
You can directly use the instanceof cast:
```suggestion
        if (newSelection instanceof IStructuredSelection iss) {
```

Reply:
done